src/wwwroot/app.js

```diff
@@ -133,7 +160,30 @@
 
     }]);
 
-  dopplerRelayModule.run(['$rootScope', 'auth', '$location', '$translate', 'jwtHelper', function ($rootScope, auth, $location, $translate, jwtHelper) {
+  dopplerRelayModule.run([
+    '$rootScope', 
+    'auth', 
+    '$location', 
+    '$translate', 
+    'jwtHelper', 
+    '$locale',
+    function (
+      $rootScope,
+      auth,
+      $location, 
+      $translate, 
+      jwtHelper,
+      $locale) {
+
+    function applyCulture() {
+      $locale.NUMBER_FORMATS.DECIMAL_SEP = $translate.instant("NUMBER_FORMATS.DECIMAL_SEP");
+      $locale.NUMBER_FORMATS.GROUP_SEP = $translate.instant("NUMBER_FORMATS.GROUP_SEP");
```

Comment:
Also, we could take advantage of our translation variables convention and avoid `$translate.instant`, we could do something like:

```js
function applyCulture() {
  var lang = $translate.use();
  var cultureFormat = window['relay-translation-' + lang]['CULTURE_FORMAT'];
  angular.extend($locale, cultureFormat);
}
```